packages/sources/lcx/src/adapter.ts

```diff
@@ -1,36 +1,41 @@
-import { Execute } from '@chainlink/types'
+import { ExecuteWithConfig, ExecuteFactory, Config} from '@chainlink/types'
 import { Requester, Validator } from '@chainlink/ea-bootstrap'
 import { util } from '@chainlink/ea-bootstrap'
+import { makeConfig } from './config'
 
 const customParams = {
   base: ['base', 'from', 'coin'],
   quote: ['quote', 'to', 'market'],
   endpoint: false,
 }
 
-export const execute: Execute = async (input) => {
-  const validator = new Validator(input, customParams)
+export const execute: ExecuteWithConfig<Config> = async (request, config) => {
+  const validator = new Validator(request, customParams)
   if (validator.error) throw validator.error
 
   const jobRunID = validator.validated.id
   const coin = validator.validated.data.base.toUpperCase()
   const currency = validator.validated.data.quote.toUpperCase()
-  const url = `https://rp.lcx.com/v1/rates/current`
+  const url = config.api.baseURL
 
   const params = {
     coin,
     currency,
   }
 
-  const config = {
+  const options = {
```

Comment:
Similar comment to ones that happened above - could clean this up by spreading in `baseUrl` and `headers` from the `config`.